content/en/methods/accounts.md

```diff
@@ -730,7 +730,8 @@ Statuses posted to the given account.
 2.6.0 - add `min_id`\
 2.7.0 - add `exclude_reblogs` and allow unauthed use\
 2.8.0 - add `tagged` parameter\
-3.3.0 - both `min_id` and `max_id` can be used at the same time now
+3.3.0 - both `min_id` and `max_id` can be used at the same time now\
+4.2.1 - add `exclude_sensitive`
```

Comment:
Let me know what the appropriate version number is here!

Reply:
typically new features don't get released in PATCH updates, so it would be the next MINOR update (4.3.0 if accepted for that release cycle)